deploy-board/deploy_board/templates/configs/new_capacity_adv.html

```diff
@@ -472,6 +474,12 @@ <h4 class="panel-title pull-left">Capacity</h4>
                 return false
             }
 
+            const replacementTimeout = clusterInfo['replacementTimeout']
+            if (replacementTimeout == null || isNaN(replacementTimeout) || replacementTimeout < 5 || replacementTimeout > 24 * 60) {
```

Comment:
looks like this check can be bypassed using API.

Reply:
The API has its own check